src/screens/MetricsScreen.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { SafeAreaView, StyleSheet, Dimensions, View } from 'react-native';
+import { Button, SegmentedButtons, Text, TextInput, ToggleButton } from 'react-native-paper';
+import { Routes } from '../routes/routes';
+import { useNavigation } from '@react-navigation/native';
+import { RouteParams } from '../routes/types';
+import { StackNavigationProp } from '@react-navigation/stack';
+// import { Colors } from '../constants';
+// import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+// import { Routes } from '../routes/routes';
+// import { RouteProp, useRoute } from '@react-navigation/native';
+// import { RouteParams } from '../routes/types';
+
+// type RouteType = RouteProp<RouteParams, Routes.Login>;
+type RoutePropType = StackNavigationProp<RouteParams, Routes.Metrics>;
+
+const MetricsScreen: React.FC = () => {
+  const [value, setValue] = React.useState('male');
+  const [selected, setSelected] = React.useState('male');
```

Comment:
What is the difference between these 2 `useStates`?

Reply:
Am uitat sa il sterg pe cel cu setValue, a ramas de la incercarea cu segmentedButtons